src/rev_ai/apiclient.py

```diff
@@ -178,26 +178,36 @@ def get_list_of_jobs(self, limit=None, starting_after=None):
 
         return [Job.from_json(job) for job in response.json()]
 
-    def get_transcript_text(self, id_):
+    def get_transcript_text(self, id_, filename = None, filepath = ''):
         """Get the transcript of a specific job as plain text.
 
         :param id_: id of job to be requested
+               filename: (Optional) filename to save text to (without extension)
```

Comment:
Our format for adding documentation for a parameter is `:param {parameterName} (optional): {description}`. Both filename and filepath should have their own :param: sections